components/eamxx/scripts/jenkins/jenkins_common_impl.sh

```diff
@@ -93,6 +93,16 @@ if [ $skip_testing -eq 0 ]; then
       fi
   fi
 
+  # AT runs may need an upstream merge in order to ensure that any DIFFs
+  # are caused by this PR and not simply because the PR is too far behind master.
+  if [ -z "$SCREAM_FAKE_ONLY" && $is_at_run == 1 ]; then
+      ./scripts/git-merge-ref origin/master
```

Comment:
Do the merge here so that test-all-scream doesn't have to worry about it.

Reply:
I think we are now handling the merge inside the jenkins plugin, so we may as well remove it here... One less thing to worry about in the scripts...

Reply:
oh, excellent